Tasks/AzureRmWebAppDeployment/XdtTransformation.ps1

```diff
@@ -0,0 +1,40 @@
+$XmlTranformDllPath = "$PSScriptRoot\Microsoft.Web.Xdt\Microsoft.Web.XmlTransform.dll"
+[Reflection.Assembly]::LoadFile($XmlTranformDllPath)
+
+function FindAndApplyTransformation
+{
+    Param(
+        [String][Parameter(mandatory=$true)]
+        $baseFile,
+        [String][Parameter(mandatory=$true)]
+        $tranformFile,
+        [String][Parameter(Mandatory=$true)]
+        $xdtFilesRoot
+    )
+
+    $tranformFiles = Find-VstsFiles -LegacyPattern "$xdtFilesRoot\**\$tranformFile" -IncludeFiles
+
+    if($tranformFiles.Count -ne 1)
+    {
+        Throw (Get-VstsLocString -Key "Noormorethanonetranformationfile0found" -ArgumentList $tranformFile)
+    }
+
+    $baseDocument = New-Object -TypeName Microsoft.Web.XmlTransform.XmlTransformableDocument
+    $baseDocument.PreserveWhitespace = $true
+    $baseDocument.Load($baseFile)
+
+    $tranformationDocument = New-Object -TypeName Microsoft.Web.XmlTransform.XmlTransformation -ArgumentList $tranformFiles
```

Comment:
When there is more than one file, what is the order in which transformation happens?